javarosa/src/test/java/org/javarosa/core/model/test/FormDefTest.java

```diff
@@ -212,7 +212,7 @@ public void testNestedRepeatActions() throws Exception {
         stepThroughEntireForm(fec);
 
         ExprEvalUtils.assertEqualsXpathEval("Nested repeats did not evaluate to the proper outcome",
-                30.0,
+                60.0,
```

Comment:
This test change expands the repeats out far enough that it triggers the bug. Previously the outer repeat count was always smaller than the inner repeat count.